e2e_tests/provider_cfg/trusted-service/Dockerfile

```diff
@@ -8,14 +8,19 @@ RUN apt-get update && \
 	apt-get install -y clang libclang-dev && \
 	# Needed for Open SSL
 	apt-get install -y pkg-config libssl-dev python3-distutils && \
-	apt-get install -y python3-distutils python3-protobuf nanopb
+	# Needed for Protobuf handling
+	apt-get install -y python3-distutils python3-protobuf nanopb && \
+	# Needed for the Trusted Services implementation
+	apt-get install -y libc++-dev libc++abi-dev 
 
 WORKDIR /tmp
-COPY trusted-services /tmp/trusted-services
-RUN cd trusted-services/deployments/ts-lib/posix-x86/ \
+RUN git clone https://git.trustedfirmware.org/TS/trusted-services.git --branch integration
```

Comment:
Is it needed if the code is already included in Parsec as a submodule?

Reply:
Well, then it would have to be part of the CI script - I'd rather it's here, no big loss.

Reply:
Actually, I guess I could put these steps in a different script or in commands in the YAML CI file